src/ApiRoute.php

```diff
@@ -133,32 +132,24 @@ public function setAction(string $action, string $method = null): void
 		$this->actions[$method] = $action;
 	}
 
-
-	private function prepareForMatch(string $string): string
-	{
-		return sprintf('/%s/', str_replace('/', '\/', $string));
-	}
-
-
 	/**
 	 * Get all parameters from url mask
 	 */
 	public function getPlacehodlerParameters(): array
 	{
-		if (!empty($this->placeholder_order)) {
+		if ($this->placeholder_order) {
```

Comment:
I do not think this will pass phpstan.. Could you pls change to `if ($this->placeholder_order !== []) {`?